src/views/ProjectView.vue

```diff
@@ -122,9 +124,18 @@ const sessionNameExists = (name) => {
     return uniqueDataSessions.value.some(session => session.name === name)
 }
 
+// invoking loadConfig when view first mounts
+onMounted(async () => {
+  const config = await loadConfig()
+  apiBaseUrl = config.dataSessionApiUrl + '/datasessions/'
+  // for rendering purposes
+  isConfigLoaded.value = true
+})
+
 </script>
 <template>
-    <div class="container">
+    <!-- only load if config is loaded -->
+    <div v-if="isConfigLoaded" class="container">
```

Comment:
If you load the config file on Vue App load, you will not need to do this here or in any other components